crates/cxx-qt-gen/src/parser/signals.rs

```diff
@@ -34,10 +33,16 @@ pub struct ParsedSignal {
     pub inherit: bool,
     /// Whether the signal is private
     pub private: bool,
-    /// All the doc attributes (each line) of the Signal
+    /// All the doc attributes (each line) of the signal
     pub docs: Vec<Attribute>,
 }
 
+impl Invokable for &ParsedSignal {
+    fn name(&self) -> &Name {
+        &self.name
+    }
+}
+
```

Comment:
A whole trait seems a bit overkill. Maybe prefer using a lambda in the structuring phase instead that just accesses the right member.

Reply:
Done